docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -278,7 +278,40 @@ External table indexes must be a subset of the indexes of the current table, or
 
 ##### Using an index
 
-Important: "_index_" refers to one digit of the index part of the row OID. Example, if the column OID is `1.2.3.1.2` and the row OID is `1.2.3.1.2.7.8.9`, the full index is `7.8.9`. In this example, when using `index: 1`, we will refer to `7`, `index: 2` will refer to `8`, and so on.
+Important: "_index_" refers to one digit of the index part of the row OID. For example, if the column OID is `1.2.3.1.2` and the row OID is `1.2.3.1.2.7.8.9`, the full index is `7.8.9`. In this example, `index: 1` refers to `7` and `index: 2` refers to `8`, and so on.  
+
+Here is specific example of an OID with multiple positions in the index ([OID ref](http://oidref.com/1.3.6.1.4.1.5.1.1.19.25.1)):
+
+```
+accDnForwardFilterEntry OBJECT-TYPE
+         SYNTAX   AccDnForwardFilterEntry
+         ACCESS   not-accessible
+         STATUS   mandatory
+         DESCRIPTION
+             " A single set of messages that the Network Manager
+               wants selectively treated."
+         INDEX { accDnForwardFilterDest, 
+                 accDnForwardFilterSource }
+         ::= { accDnForwardFilterTable 1 }
+```
+
+The index in the case is a combination of `accDnForwardFilterDest` and `accDnForwardFilterSource`. Inspecting the `OBJECT-TYPE` of `accDnForwardFilterDest` reveals the `SYNTAX` as `INTEGER` ([OID ref](http://oidref.com/1.3.6.1.4.1.5.1.1.19.25.1.1)):
+
+```
+accDnForwardFilterDest OBJECT-TYPE
+             SYNTAX   INTEGER (0..65535)
+             ACCESS   read-write
+             STATUS   mandatory
+            DESCRIPTION
+                 " If 0, all DECNET Addresses.
+                   If an Area with Host Part  0,  all  traffic  to
+                   that Area.
+                   If a DECNET Host ID, all traffic to that Host."
+            ::= { accDnForwardFilterEntry 1 }
+```
+The value corresponding to the `accDnForwardFilterDest` of a row in this table will be found at the following OID: `.1.3.6.1.4.1.5.1.1.19.25.1.1.<accDnForwardFilterDest>`.  Retrieving the value of `.1.3.6.1.4.1.5.1.1.19.25.1.1.123` will return `123`.  To capture the value `123` and apply it as a metric tag, target `index: 1`.  
+
+The value corresponding to the `accDnForwardFilterSource` of a row in this table will be found at the following OID: `.1.3.6.1.4.1.5.1.1.19.25.1.1.<accDnForwardFilterDest>.<accDnForwardFilterSource>`.  Retrieving the value of `.1.3.6.1.4.1.5.1.1.19.25.1.1.123.456` will return `456`.  To capture the value `456` and apply it as a metric tag, target `index: 2`.  
```

Comment:
I think it could more clear to illustrate with an example like this one:

```suggestion
metrics:
  - MIB: CISCO-FIREWALL-MIB
    table:
      OID: 1.3.6.1.4.1.9.9.147.1.2.2.2
      name: cfwConnectionStatTable
    symbols:
      - OID: 1.3.6.1.4.1.9.9.147.1.2.2.2.1.5
        name: cfwConnectionStatValue
    metric_tags:
      - index: 1 // capture first index digit
        tag: service_type
      - index: 2 // capture second index digit
        tag: stat_type
```

from a real example here https://github.com/DataDog/integrations-core/blob/3a7b1d1877b7a0dfcd16e2ff856e636c8717ef5b/snmp/datadog_checks/snmp/data/profiles/_cisco-asa.yaml#L7-L18

For example, when we fetch the value of `cfwConnectionStatValue`, the OID with the index is like `1.3.6.1.4.1.9.9.147.1.2.2.2.1.5.20.2 = 4087850099`, here the indexes are `20.2` (`1.3.6.1.4.1.9.9.147.1.2.2.2.1.5.<service type>.<stat type>`).

```
cfwConnectionStatEntry OBJECT-TYPE
        SYNTAX     CfwConnectionStatEntry
        MAX-ACCESS not-accessible
        STATUS     current
        DESCRIPTION
            "An entry in the table, containing information about a
            firewall statistic."
        INDEX { cfwConnectionStatService, cfwConnectionStatType }
    ::= { cfwConnectionStatTable 1 }
```

Reply:
Sounds good, thanks @AlexandreYang, pushed up a commit with these updates: https://github.com/DataDog/integrations-core/pull/12438/commits/192ff2226ec82e69e2dd07c103ab0ef7e5b99026

Let me know what you think!

Reply:
awesome -- sure thing!  One more thing here @AlexandreYang it looks like I am not authorized to merge this PR - would you be able to do so on or point me in the right direction for how I can obtain authorization? [Screenshot showing `Merging is blocked` auth message](https://a.cl.ly/ApuXem6N)